app/src/test/java/com/futurice/freesound/feature/search/DefaultSearchDataModelTest.java

```diff
@@ -127,10 +128,12 @@ public void getSearchResultsOnceAndStream_hasNoTerminalEvent() {
 
     @Test
     public void getSearchErrorOnceAndStream_hasNoneAsDefault() {
-        defaultSearchDataModel.getSearchErrorOnceAndStream()
-                              .test()
-                              .assertNotTerminated()
-                              .assertValue(Option.none());
+        assertEquals(defaultSearchDataModel.getSearchStateOnceAndStream()
+                                           .test()
+                                           .assertNotTerminated()
+                                           .values()
```

Comment:
Split this up into different assertions (don't nest them). Also use assertJ assertions, rather than `junit.framework.Assert.*`